src/test/com/gd/twitterstreamingtToDatalake/ConfigValidationTest.scala

```diff
@@ -0,0 +1,31 @@
+package com.gd.twitterstreamingtToDatalake
+
+import org.scalatest.FunSuite
+
+class ConfigValidationTest extends FunSuite {
```

Comment:
This comment is regarding the class name or you want me to switch to another test suite FlatSpec ?

Reply:
Sorry for confusion, I didn't communicate my concern properly.

So, yes, let's move to a Spec.
Let's use `FreeSpec` in particular, since it's utilized on the account.